crate/server/src/database/sqlite/instance.rs

```diff
@@ -16,7 +17,7 @@ pub(crate) struct Sqlite<const WORD_LENGTH: usize> {
     pub(crate) pool: Pool,
 }
 
-pub use cosmian_findex::FINDEX_TABLE_NAME as FINDEX_MEMORY_TABLE_NAME;
+pub const FINDEX_MEMORY_TABLE_NAME: &str = "findex_memory";
 pub const FINDEX_PERMISSIONS_TABLE_NAME: &str = "findex_permissions";
 pub const FINDEX_DATASETS_TABLE_NAME: &str = "findex_datasets";
```

Comment:
There is no such thing as a "Findex dataset": Findex is an index, not a dataset. Also why plural here?
```suggestion
pub const DATASET_TABLE_NAME: &str = "dataset";
```

Reply:
There is a global (company-wise) confusion about what Findex is: it is both a product and a technology.

I am okay that whatever related to securing databases be marketed as Findex, however, as developer, being precise is paramount. I am strongly in favor of only using the term Findex when referring to the specific SSE scheme designed at Cosmian, and published in an eponymous paper. As such, dataset, permissions and memories are *not* related to Findex. A global review of the naming may be a good thing. The very term Findex Server may also need to be reviewed such that it does not conflict with changing the SSE algorithm the day we want to use PLOC. Thinking about it, the whole server may be designed by isolating the SSE part such that changing the SSE scheme used does not involve huge refactoring. I don't know this code base enough to tell if this is the case or not.

Reply:
Also, it may be easier to manage the permission through the SSE scheme which implements an index (map from keywords to sets of values). This is quite involving and may therefore be deferred to the next release. Since all SSE schemes designed at Cosmian should be database independent, this would avoid rewriting per-database functions to manage permissions.